app/models/course_instance.rb

```diff
@@ -18,6 +18,11 @@ class CourseInstance < ActiveRecord::Base
     .unenrolled(user)
   }
 
+  def schedule
+    return 'TBA' if meetings.blank?
+    meetings.map(&:schedule).join '; '
```

Comment:
No need to actually change it now that you've merged, but I just remembered the fancy way to do this in one line.
`meetings.map(&:schedule).join('; ').presence || 'TBA'`

`.presence` returns nil if the object would have responded `false` to `.present?` (opposite of `.blank?`), and otherwise it returns the object itself.